cobc/ppparse.y

```diff
@@ -1536,7 +1538,7 @@ copy_source:
   TOKEN
   {
 	$$ = fix_filename ($1);
-	if (cb_fold_copy == COB_FOLD_LOWER) {
+	if (cb_fold_copy == COB_FOLD_LOWER || cb_fold_copy == COB_FOLD_BOTH) {
```

Comment:
if both is added it should uppercase first, then lowercase (and actually this is what the commit log, says :-).
...

Is the folding both a GCOS related issue? Please elaborate.